web/html/src/components/FormulaForm.js

```diff
@@ -73,6 +73,12 @@ class FormulaForm extends React.Component {
                     metadata: {}
                 });
             else {
+                if (data.formula_list.filter(formula => formula == data.formula_name).length > 1) {
+                    this.state.errors.push(
+                        'Multiple Group formulas detected. Only one formula for "'
+                            + capitalize(data.formula_name) + '" can be used on each system!'
```

Comment:
All messages should be localized like below:
```suggestion
                        t('Multiple Group formulas detected. Only one formula for "{0}" can be used on each system!', capitalize(data.formula_name))
```

Reply:
Forgot about localization :see_no_evil:, thanks!